python/ray/data/dataset.py

```diff
@@ -2279,7 +2279,9 @@ def to_mars(self) -> "mars.DataFrame":
         Returns:
             A MARS dataframe created from this dataset.
         """
-        raise NotImplementedError  # P1
+        import mars
+
+        return mars.read_ray_dataset(self)
```

Comment:
Can you add unit tests for this in data/tests?

Reply:
added `test_mars.py`